server.js

```diff
@@ -1,33 +1,41 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import books from './data/books.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const port = process.env.PORT || 8000
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hi everyone! Keep reading here...COMING SOON')
+})
+
+app.get('/books', (req, res) => {
+  const { page } = req.query
+  const startIndex = 20 * +page
+  res.json(books.slice(startIndex, startIndex + 20))
+})
```

Comment:
It would be nice to make `page` 0 by default so your users get data when they didn't provide the `page` number.